device_plugins/gpu_device_plugin.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: deviceplugin.intel.com/v1
+kind: GpuDevicePlugin
+metadata:
+  name: gpudeviceplugin-sample
+spec:
+  image: registry.connect.redhat.com/intel/intel-gpu-plugin@sha256:29d686b6cda943aa5814b23aabc9709983659613be61aed288e30b7ec579b767
+  initImage: registry.connect.redhat.com/intel/intel-gpu-initcontainer@sha256:a0a2422d5c4fb42947b2b69e353800b212cb221d6bbdfb7a5ed446258998df89
+  sharedDevNum: 10
```

Comment:
Please change the shareDevNum as 1. We only support 1 915 resource :1 pod mode in this release.
And also make sure preferredAllocationPolicy is set properly.
Thanks!

Reply:
Done